stdlib/public/core/Bool.swift

```diff
@@ -294,3 +294,19 @@ extension Bool {
     return lhs ? true : try rhs()
   }
 }
+
+extension Bool {
+  @_inlineable // FIXME(sil-serialize-all)
```

Comment:
I think you can drop the `FIXME(sil-serialize-all)` for this one. It's truly legitimately inlineable.